### PR TITLE
Duration fix

### PR DESCRIPTION
**Pull request by bpogy**
- MT-2670 Fix for duration format error: "Duration must be in the format: P[nD][T[nH][nM][nS]]"

### CODE REVIEW

File: mpd/duration.go [hunk 1]
```diff
@@ -54,29 +54,16 @@ func (d *Duration) String() string {
 	if u < uint64(time.Second) {
```
Comment (JackDunham) [inline mpd/duration.go]: Should this be LTE, rather than strictly LT, given that line 59 handles the strictly-equal case?

File: go.mod [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module github.com/zencoder/go-dash/v3
 
-go 1.16
```
Comment (JackDunham) [inline go.mod:3]: Maybe just leave at 1.16...or no greater than 1.18 (for bakery compatibility).

**Review by JackDunham — APPROVED** [review]
Couple of comment/question, otherwise LGTM.

File: mpd/duration.go (hunk 1 above)
Comment (bpogy) [reply mpd/duration.go]: 0 seconds ends-up  to be < uint64(time.Second). There is unit test for that scenario, but I'll change it.

File: go.mod (hunk 2 above)
Comment (bpogy) [reply go.mod:3]: back to 1.16

**Pull request merged**